src/Hazelcast.Net/Clustering/MemberConnection.cs

```diff
@@ -142,8 +145,14 @@ public Action<ClientMessage> ReceivedEvent
 
         /// <summary>
         /// Gets the network address the client is connected to.
+        /// <para>In case of unisocket mode, gets the <see cref="RemoteAddress"/></para>
```

Comment:
Correction:
In case of unisocket mode and hazelcast cloud is not enabled, gets the <